src/languages/en.ts

```diff
@@ -2004,6 +2005,8 @@ export default {
             welcomeNote: ({workspaceName}: WelcomeNoteParams) =>
                 `You have been invited to ${workspaceName || 'a workspace'}! Download the Expensify mobile app at use.expensify.com/download to start tracking your expenses.`,
             subscription: 'Subscription',
+            markAsExported: 'Mark as manually entered',
+            exportIntegrationSelected: ({integrationName}: {integrationName: string}) => `Export to ${integrationName}`,
```

Comment:
We use this approach in all QBO/Xero/NetSuite translations, let's make it a standard
```suggestion
            exportIntegrationSelected: (integrationName: ConnectionName) => {
                const userFriendlyIntegrationName =
                    currentIntegration && CONST.POLICY.CONNECTIONS.NAME_USER_FRIENDLY[currentIntegration]
                        ? CONST.POLICY.CONNECTIONS.NAME_USER_FRIENDLY[currentIntegration]
                        : 'this integration';
                return `Export to ${userFriendlyIntegrationName}`
            },
```